src/dsl/cb.rs

```diff
@@ -209,27 +269,49 @@ pub fn if_next_step<F: Clone, T: Into<Constraint<F>>>(
     }
 }
 
+/// The **`next_step_must_be`** function takes a **`StepTypeHandler`** and returns a new constraint that requires the next step to be of the given step type.
+/// # **Arguments:**
+/// - **`step_type`**: A **`StepTypeHandler`** object.
+/// # **Return:**
+/// A **`Constraint<F>`** representing the requirement that the next step must be of the given step type.
 pub fn next_step_must_be<F: From<u64>>(step_type: StepTypeHandler) -> Constraint<F> {
     annotate(
         format!("next_step_must_be({})", step_type.annotation),
         not(step_type.next()),
     )
 }
 
+/// The **`next_step_must_not_be`** function takes a **`StepTypeHandler`** and returns a new constraint that requires the next step to not be of the given step type.
+/// # **Arguments:**
+/// - **`step_type`**: A **`StepTypeHandler`** object.
+/// # **Return:**
+/// A **`Constraint<F>`** representing the requirement that the next step must not be of the given step type.
 pub fn next_step_must_not_be<F: From<u64>>(step_type: StepTypeHandler) -> Constraint<F> {
     annotate(
         format!("next_step_must_be({})", step_type.annotation),
         step_type.next(),
     )
 }
 
+/// The **`annotate`** function takes a string annotation and an expression, and returns a new constraint with the given annotation and expression.
```

Comment:
We can omit the subject in the sentence because the document will render them. Removing the subject can make the document look cleaner
Before:
![Screenshot 2023-05-03 at 2 47 14 PM](https://user-images.githubusercontent.com/3391420/235849159-41c2f172-1680-43cb-801c-81f83c6b0328.png)
After:
![Screenshot 2023-05-03 at 2 49 01 PM](https://user-images.githubusercontent.com/3391420/235849196-e3aca5be-6fe1-4b70-9329-d2d1b9943272.png)


```suggestion
/// Take a string annotation and an expression, and returns a new constraint with the given annotation and expression.
```

Reply:
Fixed in pr#18